Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/Render/MarkdownRenderer.Inlines.cs

```diff
@@ -0,0 +1,493 @@
+            {
+                throw new RenderContextIncorrectException();
+            }
+
+            Span span = new Span();
+
+            if (TextDecorationsSupported)
+            {
+                span.TextDecorations = TextDecorations.Strikethrough;
+            }
+            else
+            {
+                span.FontFamily = new FontFamily("Consolas");
+            }
+
+            var childContext = new InlineRenderContext(span.Inlines, context)
+            {
+                Parent = span
+            };
+
+            // Render the children into the inline.
+            RenderInlineChildren(element.Inlines, childContext);
+
+            if (!TextDecorationsSupported)
+            {
+                AlterChildRuns(span, (parentSpan, run) =>
+                {
+                    var text = run.Text;
+                    var builder = new StringBuilder(text.Length * 2);
+                    foreach (var c in text)
+                    {
+                        builder.Append((char)0x0336);
+                        builder.Append(c);
+                    }
+                    run.Text = builder.ToString();
+                });
+            }
+
+            // Add it to the current inlines
+            localContext.InlineCollection.Add(span);
+        }
+
+        /// <summary>
+        /// Renders a superscript element.
+        /// </summary>
+        /// <param name="element"> The parsed inline element to render. </param>
+        /// <param name="context"> Persistent state. </param>
+        protected override void RenderSuperscriptRun(SuperscriptTextInline element, IRenderContext context)
+        {
+            var localContext = context as InlineRenderContext;
+            var parent = localContext?.Parent as TextElement;
+            if (localContext == null && parent == null)
+            {
+                throw new RenderContextIncorrectException();
+            }
+
+            // Le <sigh>, InlineUIContainers are not allowed within hyperlinks.
+            if (localContext.WithinHyperlink)
+            {
+                RenderInlineChildren(element.Inlines, context);
+                return;
+            }
+
+            var paragraph = new Paragraph
+            {
+                FontSize = parent.FontSize * 0.8,
+                FontFamily = parent.FontFamily,
+                FontStyle = parent.FontStyle,
+                FontWeight = parent.FontWeight
+            };
+
+            var childContext = new InlineRenderContext(paragraph.Inlines, context)
+            {
+                Parent = paragraph
+            };
+
+            RenderInlineChildren(element.Inlines, childContext);
+
+            var richTextBlock = CreateOrReuseRichTextBlock(new UIElementCollectionRenderContext(null, context));
+            richTextBlock.Blocks.Add(paragraph);
+
+            var border = new Border
+            {
+                Padding = new Thickness(0, 0, 0, paragraph.FontSize * 0.2),
```

Comment:
Magic Number?

Reply:
I didn't change anything here, it was moved.

Reply:
As discussed offline, was here before, we can worry about another day.

Reply:
This was an existing implementation.